content-team/working-docs/social-media-policy.md

```diff
@@ -0,0 +1,90 @@
+# Social Media Strategy and Best Practices
+DD&I Content Team
+
+## Channels
+- [Twitter](https://twitter.com/drupaldiversity, "Drupal Diversity Twitter")
+- [Instagram](https://www.instagram.com/drupaldiversity/, "Drupal Diversity Instagram" )
+- [Drupaldiversity.com blog](https://drupaldiversity.github.io/, "Drupal Diversity Blog")
+- Newsletter 
+
+## Purpose
+*What are we attempting to achieve with our social media?*
+
+- To cultivate a more inclusive Drupal community.
+- To give recognition to the diverse voices we have.
+- Transparency, public participation and collaboration.
+
+## Audience
+*Who are we trying to reach?*
+
+- For the community.
+- For the volunteers who contribute to our issue queue.
+- For mentors, and teachers in teh Drupal community.
+- For people who organize and attend Drupal events.
+
+## Content
+*What types of content do we post? How does that content support our mission? What do our audiences want to hear?*
+
+We want to post content that is related to diversity (age/race/gender/religion/more) and/or tech and various places where these issues cross over. Sharing content with different perspectives and difference voices allows us to learn and grow in our understanding.
```

Comment:
I suggest changing diversity to *diversity and inclusion*.

I agree with including D&I *and/or* tech, as well items that touch both areas.